### ISSUE TITLE
Rendering view stops laravel_session cookie from being sent

### ISSUE THREAD

**Issue by cyrrill** [laravel/framework#9802]
In a controller:

```
public function index()
{
    return view('large_view_with_includes_and_sections');
}
```

When hitting this page, the output is rendered correctly in the browser, there are no exceptions or even console errors in the browser to indicate any issue at all, it looks as expected. However, the laravel_session is not sent by Laravel. Inspecting the network traffic shows no Set-Cookie header is passed down.

If I replace that view for:

```
public function index()
{
    return view('small_test_view_saying_helloworld');
}
```

The "hello world" text is output as expected, and the laravel_session cookie is sent correctly. So this shows there is nothing wrong with cookies, or server setup.

I suspect there is something in the view which is causing this issue. However, there are no errors or anything in the laravel, php or nginx logs to say something is wrong; plus the original view renders as expected.

So either some error message should show up somewhere (preferably an exception), or there should be no reason for the view rendering to interfere with the cookies being sent.

**Comment by cyrrill** [comment]
Is it possible that a Blade syntax error or similar could silently fail and somehow stop the cookie being sent?

**Comment by cyrrill** [comment]
@GrahamCampbell 

Why close the issue without any further analysis? 

Digging deeper into the issue, I found what appears to be causing this issue.

I commented out all the Blade tags, and then added them back in progressively. I found that when using the translation function, if the key has 4 nested levels, it will fail, 3 or less works:

```
{{ trans('app.page.title') }}
```

works

```
{{ trans('app.page.section.key') }}
```

fails.

The strange thing is that when you put that directly into the controller, it works as expected, but when included in a Blade template, it stops the cookies from being sent. I know it sounds odd, but this happening.
